packages/user-interface/src/components/Layout.tsx

```diff
@@ -15,20 +9,41 @@ import ResponsiveContent from "@gemeente-denhaag/responsive-content";
 import classNames from "classnames";
 import Header from "./Header";
 import Menu from "./Menu";
-import { PortalPage } from "../interfaces/portal-page";
 import { PortalFooter } from "../interfaces/portal-footer";
-import Page from "./Page";
 import Footer from "./Footer";
 import LayoutProvider from "../providers/LayoutProvider";
 import UserInformationProvider from "../providers/UserInformationProvider";
-import LinkToParent from "./LinkToParent";
-import OfflinePage from "../pages/OfflinePage";
 import FormIoUploader from "./FormIoUploader";
 import styles from "./Layout.module.scss";
 import { HelmetProvider } from "react-helmet-async";
+import { Outlet } from "react-router-dom";
+import PageMetaData from "./PageMetaData";
+
+export interface Navigation {
```

Comment:
Zou deze niet uit de menu-items kunnen komen?

Reply:
menu-items zit in de app package, maar je hebt deze interface ook nodig in de user-interface package. En de user-interface kan geen dependency krijgen op app. Maar ik zou de interface wel in een losse file kunnen zetten zodat het layout component wat overzichtelijker blijft.